spec/controllers/logs/uploads_controller_spec.rb

```diff
@@ -47,7 +47,7 @@
 
       it 'creates log entries' do
         expect do
-          Sidekiq::Testing.inline! do
+          with_inline_sidekiq do
```

Comment:
In this PR, in `with_inline_sidekiq`, we are wrapping the block with `Prosopite.pause`, which is why we switch to it here.